### PR TITLE
bd_fix

### PR DESCRIPTION
**Pull request by BrilStrawhat**

### CODE REVIEW

File: server/src/requests/add_user_to_bd.c [hunk 1]
```diff
@@ -3,17 +3,17 @@
 static gchar *push_login(sqlite3 *db, cJSON *root, gchar *password) {
     gchar *phone = cJSON_GetObjectItem(root, "phone")->valuestring;
     gchar *query = "INSERT INTO users_credential(phone, auth_token) \
-                   VALUES(?, ?);";
+                    VALUES(?, ?);";
```
Comment (Hvvang) [inline server/src/requests/add_user_to_bd.c:6]: Delete extra space

File: server/src/requests/add_user_to_bd.c [hunk 2]
```diff
@@ -25,25 +25,25 @@ static gint push_profile_info(gint user_id, gchar *name,
                               gchar *last_name, gchar *email) {
     sqlite3 *db = *mx_get_db();
     gchar *query = "INSERT INTO user_profile(user_id, name, last_name, email) \
-                   VALUES(?, ?, ?, ?);";
+                    VALUES(?, ?, ?, ?);";
```
Comment (Hvvang) [inline server/src/requests/add_user_to_bd.c:28]: Again

**Review by Hvvang — APPROVED** [review]

File: server/src/requests/add_user_to_bd.c (hunk 1 above)
Comment (BrilStrawhat) [reply server/src/requests/add_user_to_bd.c:6]: Why?

File: server/src/requests/add_user_to_bd.c (hunk 2 above)
Comment (BrilStrawhat) [reply server/src/requests/add_user_to_bd.c:28]: Why?

**Pull request merged**